crates/sui-indexer/src/apis/read_api.rs

```diff
@@ -231,6 +234,16 @@ impl ReadApiServer for ReadApi {
     async fn get_chain_identifier(&self) -> RpcResult<String> {
         self.get_chain_identifier().await.map(|id| id.to_string())
     }
+
+    async fn verify_zklogin_signature(
+        &self,
+        _bytes: String,
+        _signature: String,
+        _intent_scope: ZkLoginIntentScope,
+        _author: SuiAddress,
+    ) -> RpcResult<ZkLoginVerifyResult> {
+        todo!()
```

Comment:
confirming i do not need to add this to indexer api?

Reply:
Correct, fine to leave this as we don't use this in production anymore -- but let's have it return an error rather than panic.